src/flow.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# Copyright (c) 2021-2024, Auburn University
+# Part of azplugins, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd import _hoomd
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.operation import _HOOMDBaseObject
+from . import _azplugins
```

Comment:
This also shouldn't be allowed by ruff, I think.
```suggestion
from hoomd.azplugins import _azplugins
```